oss-vienna-happy-hour.mdx

```diff
@@ -0,0 +1,232 @@
+---
+title: 'Postgres Happy Hour with Xata at Open Source Summit Europe'
+description: 'Join us for the Postgres Happy Hour in Vienna on September 18th!'
+image:
+  src: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/postgres-happy-hour.png
+  alt: Postgres Happy Hour with Xata at Open Source Summit Europe
+author: Gulcin Yildirim Jelinek
+authorEmail: gulcin@xata.io
+date: 09-03-2024
+tags: ['postgres', 'oss', 'vienna', 'happy-hour', 'xata', 'postgresql', 'postgres-happy-hour', 'ossumit', 'booth']
+published: true
+slug: oss-vienna-happy-hour
+ogImage: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/postgres-happy-hour.png
+---
+
+## Postgres Happy Hour with Xata at Open Source Summit Europe
+
+On September 16-18th, we'll be at the [Open Source Summit Europe](https://events.linuxfoundation.org/open-source-summit-europe/) in Vienna, Austria.
+
+We will have a booth in the showcase area on the ground floor where we'd love to chat with you about Postgres and Xata.
+Personally, this event is extra special for me, as it will be my first time doing booth duty as a new Xata employee.
+If you're attending the Open Source Summit, please stop by our booth and say hello!
+
+We are open source enthusiasts here at Xata, and here are a few of our projects:
+
+- [pgroll](https://github.com/xataio/pgroll): An open source PostgreSQL tool for zero-downtime, reversible schema migrations.
```

Comment:
Maybe for this section lets instead link to our OSS feature pages on our website and not Github to help with SEO?

Reply:
Replaced URLs with website URLs instead of Github repositories.